index.bs

```diff
@@ -296,24 +292,24 @@ partial interface NeuralNetworkContext {
     therefore its usage is encouraged from the performance standpoint.
     <pre highlight="js">
     let shape = [1,-1,1,1];
-    return nn.add(
-        nn.mul(
-          nn.reshape(scale, shape),
-          nn.div(
-            nn.sub(input, reshape(mean, shape)),
-            nn.sqrt(nn.add(nn.reshape(variance, shape), nn.constant(epsilon)))
+    return builder.add(
+        builder.mul(
+          builder.reshape(scale, shape),
+          builder.div(
+            builder.sub(input, reshape(mean, shape)),
```

Comment:
should it be `builder.reshape`? it was an old issue.

Reply:
Fixed.